kalite/topic_tools/__init__.py

```diff
@@ -67,6 +78,20 @@ def get_topic_tree(force=False, annotate=False, channel=None, language=None, par
         TOPICS = {}
     if TOPICS.get(channel) is None:
         TOPICS[channel] = {}
+
+    if settings.DO_NOT_RELOAD_CONTENT_CACHE_AT_STARTUP and not force:
```

Comment:
Because otherwise we call `softload_json(...)` (loading the original file), then discards all that in favor of a cached version.. that didn't seem to make sense.